modules/computer/os.c

```diff
@@ -336,9 +336,9 @@ parse_os_release(void)
     if (!split)
         return NULL;
 
-    for (line = *split; *line; line++) {
-        if (!strncmp(line, "PRETTY_NAME=", sizeof("PRETTY_NAME=") - 1)) {
-            pretty_name = g_strdup(line +
+    for (line = split; *line; line++) {
```

Comment:
I can't see the problem with the original code; what is this patch trying to solve?

Reply:
So the parse_os_release did not worked for me. I figured out that the for-loop is not actually looping the lines. When you add `printf("%s \n", line)` into for-loop you can see what it is actually doing:
```
ondrej@ryzen3:~/hardinfo/build$ ./hardinfo 
NAME="Ubuntu" 
AME="Ubuntu" 
ME="Ubuntu" 
E="Ubuntu" 
="Ubuntu" 
"Ubuntu" 
Ubuntu" 
buntu" 
untu" 
ntu" 
tu" 
u" 
" 
```
After this PR it seems to be working fine:
```
ondrej@ryzen3:~/hardinfo/build$ ./hardinfo 
NAME="Ubuntu" 
VERSION="18.10 (Cosmic Cuttlefish)" 
ID=ubuntu 
ID_LIKE=debian 
PRETTY_NAME="Ubuntu 18.10" 
```

Reply:
Ah, indeed, you're right! Thanks for the fix